benchmark.py

```diff
@@ -78,6 +72,17 @@ def compare(self, predicted, matchingFunc, output_fn):
                         y_true.append(1)
                         y_scores.append(predictedEx.confidence)
                         predictedEx.matched.append(output_fn)
+
+                        # Also mark any other predictions with the
+                        # same exact predicate as matched.
+                        # This is to support packages that do conjunction
+                        # splitting, and doesn't affect the results for
+                        # packages that don't.
+                        if predictedEx.splits_conjunctions:
```

Comment:
If a package supports splitting conjunctions, we go through all the other predicted extractions with the same predicate and mark them as matched to prevent extra false positives.

Reply:
Is this a complete solution to the conjunction evaluation, or just a workaround to not get penalized when trying to split conjunction?
Either is fine, just want to make sure.

Reply:
Should be a complete solution, but it requires the package to declare it "splits conjunctions" by setting the "splits_conjunction" optional parameter on the `Extraction` constructor to `True`. Since none of the current packages do this, they default have this set to False. ClausIE actually has the ability to do this, but it seems like the `ClausIE.dat` files included with the repository were run with ClausIE with this flag turned off. So I didn't enable it for ClausIE.